recipes/librdkafka/all/conanfile.py

```diff
@@ -117,8 +111,20 @@ def package_info(self):
         self.cpp_info.components["rdkafka"].names["cmake_find_package_multi"] = "rdkafka"
         self.cpp_info.components["rdkafka"].names["pkg_config"] = "rdkafka"
         self.cpp_info.components["rdkafka"].libs = ["rdkafka"]
-        if self.settings.compiler == "Visual Studio":
-            self.cpp_info.components["rdkafka"].system_libs.extend(["crypt32", "ws2_32"])
+        if self.options.zlib:
+            self.cpp_info.components["rdkafka"].requires.append("zlib::zlib")
+        if self.options.zstd:
+            self.cpp_info.components["rdkafka"].requires.append("zstd::zstd")
+        if self.options.ssl:
+            self.cpp_info.components["rdkafka"].requires.append("openssl::openssl")
+        if self.options.sasl and self.settings.os != "Windows":
+            self.cpp_info.components["rdkafka"].requires.append("cyrus-sasl::cyrus-sasl")
+        if self.options.lz4:
+            self.cpp_info.components["rdkafka"].requires.append("lz4::lz4")
+        if self.settings.os == "Windows":
+            self.cpp_info.components["rdkafka"].system_libs = ["ws2_32", "secur32"]
+            if self.options.ssl:
```

Comment:
Does it need both `crypt32` and `openssl::openssl` when `options.ssl == True`?

Reply:
Both are needed on Windows. https://github.com/edenhill/librdkafka/blob/1eeb8df3d6e8d5e42f5c88568cdb0bb862f64c89/src/rdkafka_ssl.c#L40

Reply:
and here is a function from crypt32: https://github.com/edenhill/librdkafka/blob/1eeb8df3d6e8d5e42f5c88568cdb0bb862f64c89/src/rdkafka_ssl.c#L704

Reply:
Thanks for checking.